src/sensors/SensorManager.cpp

```diff
@@ -36,110 +36,122 @@ namespace SlimeVR
+                        else if (imu[i] == IMU_BNO055)
+                            m_Sensor[i] = new BNO055Sensor(i, firstIMUAddress, imuRotation[i]);
+                        else if (imu[i] == IMU_MPU9250)
+                            m_Sensor[i] = new MPU9250Sensor(i, firstIMUAddress, imuRotation[i]);
+                        else if (imu[i] == IMU_BMI160)
+                            m_Sensor[i] = new BMI160Sensor(i, firstIMUAddress, imuRotation[i]);
+                        else if (imu[i] == IMU_MPU6500 || imu[i] == IMU_MPU6050)
+                            m_Sensor[i] = new MPU6050Sensor(i, imu[i], firstIMUAddress, imuRotation[i]);
+                        else if (imu[i] == IMU_ICM20948)
+                            m_Sensor[i] = new ICM20948Sensor(i, firstIMUAddress, imuRotation[i]);
+                    }
+
+                    if (imu[i+1] == IMU_BNO080 || imu[i+1] == IMU_BNO085 || imu[i+1] == IMU_BNO086)
+                        secondIMUAddress = I2CSCAN::pickDevice(0x4B, 0x4A, true);
+                    else if (imu[i+1] == IMU_BNO055)
+                        secondIMUAddress = I2CSCAN::pickDevice(0x28, 0x29, true);
+                    else if (imu[i+1] == IMU_MPU9250 || imu[i+1] == IMU_BMI160 || imu[i+1] == IMU_MPU6500 || imu[i+1] == IMU_MPU6050 || imu[i+1] == IMU_ICM20948)
+                        secondIMUAddress = I2CSCAN::pickDevice(0x69, 0x68, true);
+
+                    if (secondIMUAddress == 0 || secondIMUAddress == firstIMUAddress)
+                    {
+                        m_Logger.debug("IMU %d not found", i+1);
+                    }
+                    else
+                    {
+                        foundIMU = true;
+                        m_Logger.info("IMU %d found at pins %d,%d and address 0x%02X", i+1, PIN_IMU_SDA, imuSCLPin[i/2], secondIMUAddress);
+
+                        if (imu[i+1] == IMU_BNO080 || imu[i+1] == IMU_BNO085 || imu[i+1] == IMU_BNO086)
+                            m_Sensor[i+1] = new BNO080Sensor(i+1, imu[i+1], secondIMUAddress, imuRotation[i+1], imuIntPin[i+1]);
+                        else if (imu[i+1] == IMU_BNO055)
+                            m_Sensor[i+1] = new BNO055Sensor(i+1, secondIMUAddress, imuRotation[i+1]);
+                        else if (imu[i+1] == IMU_MPU9250)
+                            m_Sensor[i+1] = new MPU9250Sensor(i+1, secondIMUAddress, imuRotation[i+1]);
+                        else if (imu[i+1] == IMU_BMI160)
+                            m_Sensor[i+1] = new BMI160Sensor(i+1, secondIMUAddress, imuRotation[i+1]);
+                        else if (imu[i+1] == IMU_MPU6500 || imu[i+1] == IMU_MPU6050)
+                            m_Sensor[i+1] = new MPU6050Sensor(i+1, imu[i+1], secondIMUAddress, imuRotation[i+1]);
+                        else if (imu[i+1] == IMU_ICM20948)
+                            m_Sensor[i+1] = new ICM20948Sensor(i+1, secondIMUAddress, imuRotation[i+1]);
```

Comment:
I know this code was already here before, but this could go for being merged with the nearly identical code above, I'd love to see this either in this PR or in a future PR